tests/common/ixia/ixia_fixtures.py

```diff
@@ -0,0 +1,141 @@
+included in this file.
+"""
+
+import pytest
+import pprint
+from common.devices import FanoutHost
+from ixnetwork_restpy import SessionAssistant, Files
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_ip(testbed):
+    """ 
+    In an Ixia testbed, there is no PTF docker. 
+    Hence, we use ptf_ip field to store Ixia API server. 
+    This fixture returns the IP address of the Ixia API server.
+
+    Args: 
+       testbed (pytest fixture): The testbed fixture.
+
+    Returns:
+        Ixia PTF server (API server) IP
+    """
+    return testbed['ptf_ip']
+
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_user(duthost):
+    """
+    Return the username of Ixia API server.
+
+    Args:
+        duthost (pytest fixture): The duthost fixture.
+ 
+    Returns:
+        Ixia PTF server (API server) username.
+    """
+    return duthost.host.options['variable_manager']._hostvars[duthost.hostname]['secret_group_vars']['ixia_api_server']['user']
+
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_passwd(duthost):
+    """
+    Return the password of Ixia API server.
+
+    Args:
+        duthost (pytest fixture): The duthost fixture.
+ 
+    Returns:
+        Ixia PTF server (API server) password.
+    """
+    return duthost.host.options['variable_manager']._hostvars[duthost.hostname]['secret_group_vars']['ixia_api_server']['password']
+
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_port(duthost):
+    """
+    Return REST port of the ixia API server.
+
+    Args:
+        duthost (pytest fixture): The duthost fixture.
+ 
+    Returns:
+        Ixia PTF server (API server) REST port.
+    """
+    return duthost.host.options['variable_manager']._hostvars[duthost.hostname]['secret_group_vars']['ixia_api_server']['rest_port']
+
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_session_id(duthost):
+    """
+    Ixia PTF can spawn multiple session on the same REST port.
+    Optional for LINUX, required for windows return the session ID.
+
+    Args:
+        duthost (pytest fixture): The duthost fixture.
+
+    Returns:
+        Ixia PTF server (API server) session id.
+    """
+    return duthost.host.options['variable_manager']._hostvars[duthost.hostname]['secret_group_vars']['ixia_api_server']['session_id']
+
+
+@pytest.fixture(scope = "module")
+def ixia_dev(duthost, fanouthosts):
+    """
+    Returns the Ixia chassis IP
+
+    Args:
+        duthost (pytest fixture): The duthost fixture. 
+        fanouthosts (pytest fixture): The fanouthosts fixture.
+
+    Returns:
+        Ixia Chassis IP.
+    """
+    result = dict()
+    ixia_dev_hostnames = fanouthosts.keys()
+    for hostname in ixia_dev_hostnames:
+        result[hostname] = duthost.host.options['inventory_manager'].get_host(hostname).get_vars()['ansible_host']
+    return result
+
+
+@pytest.fixture(scope = "function")
+def ixia_api_server_session(ixia_api_serv_ip,
+    ixia_api_serv_user, ixia_api_serv_passwd, ixia_api_serv_port,
```

Comment:
Arguments should be aligned.

Reply:
I have corrected the alignment according to the PEP-8 guidelines.